tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/partners_block.html

```diff
@@ -1,12 +1,16 @@
 {% load wagtailimages_tags %}
 
-{% if value.title %}
-    <h2 class="partners__title heading--two">{{ value.title }}</h2>
-{% endif %}
+<div class="partners streamfield__partners">
+    {% if value.title %}
+        <h2 class="partners__title heading--two">{{ value.title }}</h2>
+    {% endif %}
 
 
-<div class="partners__image-card">
-    {% for logo in value.partner_logos %}
-        {% image logo fill-100x100 format-webp alt="" %}
-    {% endfor %}
+    <div class="partners__logos">
+        {% for logo in value.partner_logos %}
```

Comment:
Would it be worth naming this something like `partner_logo` to avoid clashes in the pattern library?

Reply:
Have done that - good suggestion.